test/export/test_export.py

```diff
@@ -6140,24 +6140,26 @@ def forward(self, x):
         m = M()
         eager_result = m(*inp)
 
-        if not is_retracebility_test(self._testMethodName):
-            ep = export(M(), inp, preserve_module_call_signature=("n",))
-            with self.assertRaisesRegex(
-                ValueError,
-                "Cannot unflatten multiple calls to module n while preserving its signature",
-            ):
-                torch.export.unflatten(ep)
+        def test(ep, swap):
```

Comment:
Can you move this test to test_unfatten.py? I want to make sure it works with training IR as well. This here tests training IR + run_decomp so it is still operating on a functional IR.

Reply:
I don't like `test_unflatten.py` because in general there are more variations tested here. But good point that I should test this with state as well. I'll hold off landing until I do that / add them in.

For training IR only I could do a separate explicit test?